core/templates/components/state-directives/hint-editor/hint-editor.component.ts

```diff
@@ -117,7 +122,6 @@ export class HintEditorComponent implements OnInit, OnDestroy {
           this.contextService.getEntityType() === 'question')
       }
     };
-    this.hintMemento = null;
```

Comment:
No need to assign null, check flow.
earlier: Previous value -> null -> New value.
Now: Previous value -> New value. (No significance of null found).